discord/guild.py

```diff
@@ -2933,8 +2933,8 @@ async def ban(
         self,
         user: Snowflake,
         *,
-        delete_message_seconds: Optional[int] = None,
-        delete_message_days: Optional[Literal[0, 1, 2, 3, 4, 5, 6, 7]] = None,
+        delete_message_seconds: Optional[int] = 0,
+        delete_message_days: Optional[Literal[0, 1, 2, 3, 4, 5, 6, 7]] = 0,
```

Comment:
I feel like it would be better to keep `None` here, and change the check to `delete_message_seconds is not None and not (0 <= delete_message_seconds <= 604800)`. That way these typehints are correct and it's more clear overall imo.
```suggestion
        delete_message_seconds: Optional[int] = 0,
        delete_message_days: Optional[Literal[0, 1, 2, 3, 4, 5, 6, 7]] = 0,
```